src/components/application_manager/src/commands/mobile/speak_request.cc

```diff
@@ -134,8 +134,7 @@ void SpeakRequest::ProcessTTSSpeakResponse(
 
   if (is_result_ok) {
     result_code = mobile_apis::Result::WARNINGS;
-    return_info =
-        std::string("Unsupported phoneme type sent in a prompt").c_str();
+    return_info = "Unsupported phoneme type sent in a prompt";
```

Comment:
@LevchenkoS return_info is  const char*.. please change it..
In IsWhiteSpaceExist() const char* str  is assigned value as well ..  
Please check the file for this cons`t..